internal/eventstore/v1/models/search_query.go

```diff
@@ -9,24 +9,31 @@ import (
 )
 
 type SearchQueryFactory struct {
-	columns        Columns
-	limit          uint64
-	desc           bool
-	aggregateTypes []AggregateType
-	aggregateIDs   []string
-	sequenceFrom   uint64
-	sequenceTo     uint64
-	eventTypes     []EventType
-	resourceOwner  string
-	instanceID     string
-	creationDate   time.Time
+	columns Columns
+	limit   uint64
+	desc    bool
+	queries []*query
+}
+
+type query struct {
+	desc               bool
+	aggregateTypes     []AggregateType
+	aggregateIDs       []string
+	sequenceFrom       uint64
+	sequenceTo         uint64
+	eventTypes         []EventType
+	resourceOwner      string
+	instanceID         string
+	ignoredInstanceIDs []string
+	creationDate       time.Time
+	factory            *SearchQueryFactory
 }
 
 type searchQuery struct {
 	Columns Columns
 	Limit   uint64
 	Desc    bool
-	Filters []*Filter
+	Filters [][]*Filter
```

Comment:
why did this change?

Reply:
the v1 searches were not able to create "or" queries... the current models also have a two dimensional slice for the filters